js/main.js

```diff
@@ -0,0 +1,113 @@
+// global dom variables
+const $navbarForm = document.querySelector('.navbar-form');
+const $mainForm = document.querySelector('.home-form');
+const $resultList = document.querySelector('#result-list');
+const $searchPage = document.querySelector('.main-search');
+const $resultPage = document.querySelector('#results');
+const $navbarIcon = document.querySelector('#navbar-icon');
+const $resultsHeader = document.querySelector('.results-header');
+
+// navbar search bar function
+$navbarForm.addEventListener('submit', function (event) {
+  event.preventDefault();
+  const $search = event.target.elements.search.value;
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', `https://api.jikan.moe/v4/anime?limit=5&q=${$search}&rating=tv`);
+  xhr.responseType = 'json';
+  removeAllChildNodes($resultList);
+  xhr.addEventListener('load', function () {
+    $resultsHeader.textContent = `Search results for ${$search}`;
+    for (let i = 0; i < xhr.response.data.length; i++) {
+      $resultList.appendChild(renderEntry(xhr.response.data[i]));
+
+    }
```

Comment:
Since you also call this in your `$mainForm` event listener, I recommend turning these into a separate function, and just calling the function inside of this load event. That way you aren't repeating code multiple times